src/extension/test/vs_test_controller.ts

```diff
@@ -286,7 +286,7 @@ export class VsCodeTestController implements TestEventListener, IAmDisposable {
 			return false;
 		if ((label.startsWith("(setUp") || label.startsWith("(tearDown")) && label.endsWith(")"))
 			return false;
-		if (this.discoverer?.fileTracker.supportsPubRunTest(node.suiteData.path) === false)
+		if (this.discoverer?.fileTracker.supportsPubRunTest(node.suiteData.path) === false && !this.isForceFlutterWorkspace)
```

Comment:
I think there may be other places calling `supportsPubRunTest` that want the same behaviour as this (such as in the Test CodeLens handler, which adds the Run/Debug links above the Group/Test methods?).

Reply:
Ah it looks like `supportsPubRunTest` checks `useVmForTests` from `workspaceConfig`, so I should be able to just add a flag and use it in there as well.